pl.json

```diff
@@ -298,10 +298,10 @@
         "Options.SeatedMode.On" : "Tryb Siedzący: Tak",
         "Options.SeatedMode.Off" : "Tryb Siedzący: Nie",
         "Options.Nameplates.All" : "Nazwy Graczy: Wszystkie",
-        "Options.Nameplates.NonContacts" : "Nazwy Graczy: Nie Kontakty",
+        "Options.Nameplates.NonContacts" : "Nazwy Graczy: Nie Kontaktów",
         "Options.Nameplates.Hide" : "Nazwy Graczy: Ukryte",
 
-        "Indicator.OnlineUsers" : "Użytkownicy Online: ~{num_total} ({num_registered})",
+        "Indicator.OnlineUsers" : "Użytkownicy Online: ~{num_registered} ({num_total})",
```

Comment:
i see you tried to fix this but its  still wrong, it should be `{num_registered} (~{num_total})`